### ISSUE TITLE
[1.16.3] Modded biomes not generating with EITD enabled

### ISSUE THREAD

**Issue by bessiq** [gigaherz/EyesInTheDarkness#11]
EyesInTheDarkness-1.16.3-0.5.0.jar
Forge: 34.1.0
Relevant mods: Biomes O' Plenty 1.16.3-12.0.0.399,  Biomes You Go 1.0.7 MCV: 1.16.3

Hi there!

I've uncovered a seemingly odd interaction between your mod and a couple biome mods, where their biomes will not generate when EITD is enabled. All mods have default configs, and I tested EITD with each biome mod individually (no other mods installed). I did notice that your port to 1.16.3 occurred a couple days before this Forge update, in case it's relevant:
```
Build: 1.16.3-34.0.19 - Thu Sep 24 06:36:25 GMT 2020
	lex:
		Revive BiomeManager and BiomeLayer hooks.
```

When EITD is enabled, I get the following lines repeated hundreds of times or more when a new world is generating:
`[Server thread/WARN] [net.minecraft.world.gen.layer.Layer/]: Unknown biome id: `
and then:
`[Worker-Main-26/WARN] [net.minecraft.world.gen.layer.Layer/]: Unknown biome id:`

I also get this error repeated many times:
`[Render thread/ERROR] [net.minecraft.util.registry.DynamicRegistries/]: Error loading registry data: Unknown element name: eitd_eyes missed input: {"eitd_eyes":[]}; Unknown element name: eitd_eyes missed input: {"eitd_eyes":[]}; Unknown element name: eitd_eyes missed input: {"eitd_eyes":[]};` etc. (it continues repeating on a single line a couple hundred times)

I find this mod amusing so hopefully it can be resolved. Let me know if you'd like any further info. Thanks!

**Comment by gigaherz** [comment]
huh... I'll go take a look, I built with an older forge version.

**Comment by gigaherz** [comment]
I can't reproduce the issue locally, so I made a little change that hopefully will help. Want to test it?

[EyesInTheDarkness-1.16.3-0.5.0-1.jar.zip](https://github.com/gigaherz/EyesInTheDarkness/files/5287535/EyesInTheDarkness-1.16.3-0.5.0-1.jar.zip)

The jar is contained inside the zip, blame github.

**Comment by bessiq** [comment]
I will definitely test it in the morning. Away from my PC now. Thanks for the response!

**Comment by bessiq** [comment]
Hi,

Sorry for not responding sooner; I've been running errands all day. I tested with the jar you provided, and unfortunately the same thing happens with either BoP or BYG. Again, with no other mods running, on Forge 34.1.0. I've linked my logs below in case it's helpful, but I had to use Dropbox because that unknown biome error makes them massive.  If you like, I can remove the majority of those lines and re-link the logs. Just let me know. Thanks!

EITD+BYG: https://www.dropbox.com/s/wf9zo5gs0yyy4wn/latest-BYG.log?dl=0
EITD+BoP: https://www.dropbox.com/s/f14xanbecpyam2v/latest-BoP.log?dl=0

**Comment by gigaherz** [comment]
Hi thanks for the report, I'll keep digging.

**Comment by bessiq** [comment]
Hi there,

Modded biomes are now generating normally, and I think I had even re-enabled EITD before your updates a couple days ago. I guess it was maybe an issue with certain versions of Forge?

Anyway, the new common config isn't generating for me with versions 0.6.0-0.6.1. I get the client config and the new server config that removes the biome section, but that's it. I also haven't been able to find any of the eyes in-game, when usually they spawn very frequently.

Just wanted to let you know. Thanks!

**Comment by gigaherz** [comment]
Oops, I forgot to register the COMMON config XD

**Comment by bessiq** [comment]
Good news! The update generated the common config. Less than good news: I still can't seem to find any of these eyes anywhere!

**Comment by gigaherz** [comment]
/me headdesks
I know why. This is what I get for rushing updates without proper testing.

**Comment by gigaherz** [comment]
Okay I have released another update. 
But I discovered one thing while debugging: I'M AN IDIOT AND MOBS DON'T SPAWN IF I'M IN SPECTATOR MODE
So much wasted time. I'm almost certain it would have been working before, but in any case. I removed a hack I didn't like in the process.
Downside: the spawn rules now require a world restart (eg, exiting the save and loading it again, or restarting the dedicated server).
Verified working in my dev env.
